cmd/destroy.go

```diff
@@ -34,6 +34,7 @@ func NewDestroyCmd(globalCfg *config.GlobalImpl) *cobra.Command {
 	f.StringVar(&globalCfg.GlobalOptions.Args, "args", "", "pass args to helm exec")
 	f.IntVar(&destroyOptions.Concurrency, "concurrency", 0, "maximum number of concurrent helm processes to run, 0 is unlimited")
 	f.BoolVar(&destroyOptions.SkipDeps, "skip-deps", false, `skip running "helm repo update" and "helm dependency build"`)
+	f.BoolVar(&destroyOptions.SkipCharts, "skip-charts", false, "don't prepare charts when destroying releases")
```

Comment:
```suggestion
	f.BoolVar(&destroyOptions.SkipCharts, "skip-charts", true, "don't prepare charts when destroying releases")
```

Can we probably turn it on by default, assuming the chart preparation step would be needed only when you're going to use predelete hooks contained within charts along with `helmfile destroy`(which should be a super rare case?)

Reply:
@mumoshu we can set the default to `false` when it was introduced the first time. we only add an ability to disable prepare chart when running destroy. if you think that true is better. I will update the code.

Reply:
@yxxhero Not a strong opinion. Actually, I have mixed feelings. This is a breaking change anyway...

Reply:
so that I keep it false default. 🥇

Reply:
Sounds good! Thanks for your confirmation ☺️